compaction/compaction_manager.cc

```diff
@@ -939,8 +939,7 @@ compaction_manager::compaction_manager(config cfg, abort_source& as, tasks::task
     , _update_compaction_static_shares_action([this] { return update_static_shares(static_shares()); })
     , _compaction_static_shares_observer(_cfg.static_shares.observe(_update_compaction_static_shares_action.make_observer()))
     , _strategy_control(std::make_unique<strategy_control>(*this))
-    , _tombstone_gc_state(&_repair_history_maps)
-{
+    , _tombstone_gc_state(&_reconcile_history_maps) {
```

Comment:
This rename in the same commit as the other changes is very distracting when reviewing.